person.rb

```diff
@@ -10,6 +10,12 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rentals)
+    @rentals << rentals
+    rentals.person = self
   end
```

Comment:
- [ ] [OPTIONAL] Please note that when a rental is created it's automatically associated with the person object and inside the initialize method in the Rental class the new rental is added to the `@rentals` array in the following [line](https://github.com/Tommymarvel/School_Library/blob/770d10e1df554c20d2173bb0dcee19bde1ad036f/rental.rb#L9) so it's redundant to add the new rental to the `@rentals` array.